### DISCUSSION TITLE
Cell voltage over discharge capacity

### DISCUSSION THREAD

**Discussion by NLithium** [pybamm-team/PyBaMM#2004] [General]
I was wondering how should I setup the experiment in PyBaMM to obtain following results of decreasing capacity during cycling - at least 2000 cycles?
For example, we have a simple model and we want to solve it over experiments: 
```math
model = pybamm.lithium_ion.DFN(
    {
        "SEI": "ec reaction limited",
        "SEI film resistance": "distributed",
        "SEI porosity change": "true",
        "thermal": "lumped"
    },
)

total_cycles = 100
experiment = pybamm.Experiment(
    [
        (
            "Discharge at 1 C until 2.8 V",
            "Rest for 5 minutes",
            "Charge at 1 C until 4.1 V",
            "Hold at 4.1 V until C/10",
            "Rest for 5 minutes",
        )
    ]
    * total_cycles
)
```

And the result is "Experiment is infeasible: 'event: Zero electrolyte concentration cut-off' was triggered during 'Charge at 1 C until 4.1 V'. The returned solution only contains the first 24 cycles. Try reducing the current, shortening the time interval, or reducing the period."
How can zero electrolyze concetration cut-off error be resolved?

**Comment by brosaplanella** [comment]
You can remove the event that stops the concentration (remove it from `model.events` list). Note that there is this issue #1986 in progress which should also fix this problem.

**Reply by NLithium** [reply]
Thank you for your answer. I tried it and it is cycling longer, but capacity decay is still way too fast with current model setup.

**Reply by batiz1** [reply]
We have the similar problem as well, but with minimum voltage when trying different C-rates, and as you see in the figure the battery with the lowest C-rate has the major  loss of them. Is there maybe an issue with inverse C-rates in the code or something?
![RemainingCapacity_Crates_NT_CerDc_Crate_NT_Soc0407_C05Dc05N46](https://user-images.githubusercontent.com/102738018/168291910-e70a9c73-f7f8-40d9-8f6e-a2b3486515d3.jpg)
